.github/workflows/check-links.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Docs and Site Links
+
+on:
+  schedule:
+    - cron: '30 8 * * *'
+
+jobs:
+  check-links:
+    name: Regularly check links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check docs links with linkcheck
+        uses: filiph/linkcheck@v2.0.15+1
```

Comment:
2.0.19 is available if we want to move up to that.
https://github.com/filiph/linkcheck/releases/tag/2.0.19